### DISCUSSION TITLE
Popover width

### DISCUSSION THREAD

**Discussion by QiiFeng** [twilio-labs/paste#2003] [Q&A] [answered]
Hi, my team is looking for expanding the width of the Popover to make sure the SID number can be displayed within a line. (Pls see the screenshot below, which looks not good). The documentation shows that the Popover width is not modifiable. Wondering if Paste has added any flexibility for the Popover width?

<img width="405" alt="Screenshot 2021-11-04 at 17 27 29" src="https://user-images.githubusercontent.com/86003634/140389677-5922dc97-933a-4719-8227-4b8a5e1f7b8a.png">

**Comment by nkrantz** [comment, marked as answer]
Hi @QiiFeng, please follow along (and feel free to participate in) [this thread regarding Popover widths](https://github.com/twilio-labs/paste/discussions/1732). Thanks!
